bigtable/tests/unit/test_table.py

```diff
@@ -16,6 +16,65 @@
 import unittest
 
 
+class Test___mutate_rows_request(unittest.TestCase):
+
+    def _call_fut(self, table_name, rows):
+        from google.cloud.bigtable.table import _mutate_rows_request
+
+        return _mutate_rows_request(table_name, rows)
+
+    def test__mutate_rows_too_many_mutations(self):
+        from collections import namedtuple
+        from google.cloud.bigtable._generated.data_pb2 import Mutation
+        from google.cloud.bigtable.row import DirectRow
+        import google.cloud.bigtable.table as table_module
+        from google.cloud.bigtable.table import TooManyMutationsError
+
+        table = namedtuple('Table', ['name'])
+        table.name = 'table'
+        table_module._MAX_BULK_MUTATIONS = 3
```

Comment:
This is not a good idea, because `table_module` will continue to exist after this test and you have permanently modified it. See my suggestion above for what to do instead.